contracts/test/v0.8/dev/KeeperRegistryDev.test.ts

```diff
@@ -142,53 +142,45 @@ describe('KeeperRegistryDev', () => {
     transcoder = await upkeepTranscoderFactory.connect(owner).deploy()
     registryLogic = await keeperRegistryLogicFactory
       .connect(owner)
-      .deploy(linkToken.address, linkEthFeed.address, gasPriceFeed.address)
+      .deploy(0, linkToken.address, linkEthFeed.address, gasPriceFeed.address)
+
+    const config = {
+      paymentPremiumPPB,
+      flatFeeMicroLink,
+      blockCountPerTurn,
+      checkGasLimit,
+      stalenessSeconds,
+      gasCeilingMultiplier,
+      minUpkeepSpend,
+      maxPerformGas,
+      registryGasOverhead,
+      fallbackGasPrice,
+      fallbackLinkPrice,
+      transcoder: transcoder.address,
+      registrar: ethers.constants.AddressZero,
+    }
     registry = await keeperRegistryFactory
       .connect(owner)
       .deploy(
+        0,
```

Comment:
Can you add some tests for optimism and arbitrum payment models (while mocking the oracle contracts)

Reply:
i feel it's appropriate to test it as you mentioned. but that will mean the oracle address of arb and opt will be passed into the contract. this is unnecessary for other chains and complicates the constructor params. any thoughts?

Reply:
Ah, yes agree we should not complicate the contract for purposes of testing. There should be some ways in hardhat around this. 
e.g. thread i found https://github.com/defi-wonderland/smock/issues/47

Reply:
resolved by this [post](https://mixbytes.io/blog/how-change-bytecode-already-deployed-smart-contract).